### PR TITLE
Adding update stock functionality

### LINKED ISSUE #7: Inventory DB Update - Stock
## As an administrator
**I need** to be able to update a product's stock level
**So that** we can keep track of the stock levels in our inventory 

**Assumptions:**
* The desired Database record exists AND is active

**Acceptance Criteria:**
```
Given the modified stock data
When I query the record
The it mus return with the updated data
```

## As a developer
**I need** to create:
1. Functions in model.py to update stock levels of an existing inventory record in the DB
2. Test cases for those functions
3. Corresponding Functions in service.py to setup a resource endpoint
4. Test cases for those functions (in service.py)

**Assumptions:**
1. DB schema is available
2. models.py is created
3. service.py is created
4. test_models.py is created
5. test_service.py is created

**Acceptance Criteria:**
```
I can successfully read the newly updated inventory record
```

### PR DESCRIPTION
**Pull request by arungithub9**
Added update stock levels functionality. Tested all cases with postman. Working. Will work on further issues later today. 

Example test cases : PUT /inventory/11/new/add/2 : adds 2 stocks to product 11. Error if prod not present.
PUT /inventory/11/new/sub/1 : subtracts 2 stocks from product 11
PUT /inventory/11/new/sub/100 : tries to subtract 1000 stocks, since the result will be negative, throws error and denies the transaction.

### CODE REVIEW

File: service/service.py [hunk 1]
```diff
@@ -211,6 +223,42 @@ def delete_inventory(product_id, condition):
...
+    if not inventory:
+        raise NotFound("Inventory with product_id {} and condition {} was not found.".format(product_id, condition))
+    
+    msg_tmp = " "
+
+    if operation == "add":
+        inventory.quantity = inventory.quantity + amount
+        msg_tmp = "Added "
+    else:
+        inventory.quantity = inventory.quantity - amount
```
Comment (arahant) [inline service/service.py:252]: Can you add another logic after this statement? When the ```quantity=0``` then it should be unavailable.
```
if inventory.quantity==0:
    inventory.available=0
```

**Comment by arahant** [comment]
I think the data to update the record with is provided as a payload, in the body, and not the uri itself.
If you want to remove ```n``` stocks:
```
PUT /inventory/11
body: {
    stock_count: -n
}
```

**Comment by arungithub9** [comment]
I mean, the main purpose of this specific call is to make easy add and sub requests, this would be easier imo. I does not violate any rest principles as such, so this should be good.

File: service/service.py (hunk 1 above)
Comment (arungithub9) [reply service/service.py:252]: added

**Pull request merged**